Makefile

```diff
@@ -227,6 +227,25 @@ run-test-container: ## Run docker container that you can run nexodus in
 		--mount type=bind,source=$(shell pwd)/.certs,target=/.certs,readonly \
 		quay.io/nexodus/test:$(TEST_CONTAINER_DISTRO) /update-ca.sh
 
+.dev-container: Containerfile.dev
+	$(CMD_PREFIX) docker build -f Containerfile.dev -t quay.io/nexodus/dev:latest .
+	$(CMD_PREFIX) touch $@
+
+.PHONY: run-dev-container
+run-dev-container: .dev-container ## Run docker container that you can develop and run nexodus in
```

Comment:
Do we still need run-test-container or can this solve both use cases?

Reply:
No.. so this container has developer tooling.  I think test container should get replaced by the nexd container which is basically just an image with the nexd/nexctl runtimes.

Reply:
Rational behind having a bin/sh container with nexd copied in is for testing killing and respawning nexd.